test/controllers/language-listener/language-listener.test.js

```diff
@@ -0,0 +1,23 @@
+import '../../../src/components/community/community-link.js';
```

Comment:
I'd suggest using the lang-listener demo component or just making one on-the-fly using `defineCE`. That way the unit tests aren't relying on another component which may change, move or be removed in the future.